eos/constraint.cc

```diff
@@ -1261,6 +1264,7 @@ namespace eos
             // If specified, these options allow to specify a subset of the measurements
             unsigned begin = destringify<unsigned>(options.get("begin", "0"));
             unsigned end = destringify<unsigned>(options.get("end", stringify(dim_meas)));
+            double rescale_factor = destringify<double>(options.get("rescale-factor", "1.0"));
```

Comment:
This line might benefit from a comment :)

Reply:
Oh, yes. Fixed.